### PR TITLE
P-901: replace orml with pallet_assets

### PR DESCRIPTION
**Pull request by wangminqi**
Replace orml_token with pallet_assets.
Precompile impl of pallet_assets on Rococo is also included.

pallet_bridge and pallet_staking precompile updated since the precompile-utils change.

Should including runtime upgrade of storage clean. (do not need anymore, since only rococo which will be depreciated soon)

### CODE REVIEW

**Comment by wangminqi** [comment]
Warning: the precompile utils pallet is not available in official frontier until 0.9.43. The one we use is a temporary version downgrade to 0.9.42.

**Comment by wangminqi** [comment]
Disable integration ts test on precompile, it is just on Rococo.
Let's fix it after decimal migration solved.

File: precompiles/bridge-transfer/src/lib.rs [hunk 1]
```diff
@@ -1,103 +1,54 @@
...
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// Litentry is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
```
Comment (Kailai-Wang) [inline precompiles/bridge-transfer/src/lib.rs]: We still need this

**Review by Kailai-Wang — APPROVED** [review]
Thanks!

File: precompiles/bridge-transfer/src/lib.rs (hunk 1 above)
Comment (wangminqi) [reply precompiles/bridge-transfer/src/lib.rs]: Added back

**Pull request merged**